Code/GraphMol/SmilesParse/v2catch_tests.cpp

```diff
@@ -66,4 +66,10 @@ TEST_CASE("v2 basics") {
     auto bnd = SmilesParse::BondFromSmarts("@");
     REQUIRE(bnd);
   }
-}
\ No newline at end of file
+  {
+    auto mol = SmilesParse::MolFromSmiles("[Al+]1cccccccccc1");
```

Comment:
This should be its own `TEST_CASE`, something like this:

```suggestion
}
TEST_CASE("handling of aromatic Al in SMILES"){
  SECTION("basics") {
      auto mol = SmilesParse::MolFromSmiles("[Al+]1cccccccccc1");
```
(note, I haven't tested that and may have missed some punctuation)

Reply:
Thanks for the suggestion! I applied.